---
layout: default
title: Project - LPOR
---

<div class="container-fluid" style="background-color:#f2f2f2;">
<div class="container">
    <div class="row cont-margin">
            <div class="font-oswald" style="padding-top: 120px; padding-bottom: 30px; color: black;">
                Project: Origin and evolution of light-dependent protochlorophyllide oxidoreductases (LPORs) in bacteria<hr>
            </div>
    </div>
</div>
</div>


<div class="container">
    <div class="row cont-margin">
        <div class="container box-border-green" style="width:100%; padding: 20px;">
        <div class="col-sm-4">
            
            <div class="imageAVA">
                <div class="imageInn">
                    <img src="{{"/assets/img/projects/project_1/lpor-1.png" | prepend: site.baseurl}}" style="width:100%" alt="LPOR-tree">
                </div>
                <div class="hoverImg">
                    <img src="{{"/assets/img/projects/project_1/lpor-3.png" | prepend: site.baseurl}}" style="width:100%" alt="AAPB-LPOR-subtree">
                </div>
            </div>
        </div>
        <div class="col-sm-8">
                <br>Light-dependent protochlorophyllide oxidoreductase (LPOR) is one of the proteins used in synthesis of (bacterio)chlorophyll, the primary pigment needed for both anoxygenic and oxygenic photosynthesis. Here, we uncover complex evolution of the protein in bacteria.
                </br>
               <br>
               It was long assumed that among bacteria only cyanobacteria possess LPORs. Here, we explored the spread of LPORs among aerobic anoxygenic phototrophic bacteria (AAPB), which are older than cyanobacteria. The corresponding LPOR sequences clustered within a special group of cyanobacteria, called picocyanobacteria, uncovering a cascade of horizontal gene transfers (HGTs).
                </br>
                <br style="">
                In this project I identified LPOR sequences by scanning genomes of all sequenced bacteria, using gene markers provided evidence for classification of bacteria as AAPBs, performed phylogenetic analysis using multiple phylogenetic methods and software. I explored possibilities of different hypotheses for origin of LPORs in AAPBs using phylogenetic analysis and analysis of genomes using methods for HGT detection, taking into account also species divergence times and previously published analysis of other photosynthetic proteins.
                </br>
        </div>
        </div>
    </div>
    <div class="row cont-margin">
        <div class="col-sm-6">
        <h4 class="section-header text-green">Contributions:</h4>
        <ul class="list-mar">
        <li>Data collection: exhaustive search for LPOR sequences among all bacterial genomes available at NCBI and other databases</li>
        <li>Screening of genomes for marker genes</li>
        <li>Classification of bacteria based on published characteristics about bacteria and presence-absence of marker genes</li>
        <li>Inference of phylogenetic bacterial LPOR tree (multiple independent runs, different software and methods)</li>
        <li>Inference of phylogenetic bacterial rRNA tree (multiple independent runs)</li>
        <li>Identification of conflicts between LPOR and rRNA trees</li>
        <li>Comparison of taxonomic NCBI tree and LPOR tree</li>
        <li>Genome islands analysis of bacterial genomes as the evidence of horizontal gene transfer</li>
        <li>Analysis of bacterial diversification times</li>
        <li>Simulation of sequence evolution</li>
        <li>Proposing and testing hypotheses of LPOR origin and evolution in bacteria</li>
        <li>Manuscript writing and handling of revision</li>
        <li>Results visualisation for the manuscript</li>
        
        </ul>
      
        </div>
            <div class="col-sm-6">
                <small>
                    <div class="info-message info-red">
                        <b class="text-red">Keywords:</b> protein evolution, LPORs, horizontal gene transfer
                    </div>
                    <div class="info-message info-green">
                        <b class="text-green">Bioinformatic Tools:</b> MAFFT, HMMER, BLAST, REvolver, IQ-TREE, RAxML, TNT, MrBayes, NOTUNG, iTOL, FigTree, timetree
                    </div>
                    <div class="info-message info-green">
                        <b class="text-green">Data:</b> DNA/RNA/protein sequences, biological dataset
                    </div>
                    <div class="info-message info-green">
                        <b class="text-green">Databases:</b>  ensembl, GenBank, CAMERA, oneKP, SilVa
                    </div>
                    <div class="info-message info-blue">
                        <b class="text-blue">Tech/DEV:</b> linux, MacOS, R, bash, grid cluster, custom pipelines
                    </div>
                    
                    <div class="info-message info-yellow">
                        <b class="text-yellow">Publications:</b>
                        <ul style="padding: 5px 5px 5px 15px;">
                                <li>O. Chernomor<sup>e</sup>, L. Peters<sup>e</sup>, J. Schneidewind<sup>e</sup>, A. Loeschcke, E. Knieps-Gr&uuml;nhagen, F. Schmitz, E. von Lieres, R.J. Kutta, V. Svensson, K.-E. Jaeger, T. Drepper, A. von Haeseler, and U. Krauss (2021) <b>Complex evolution of light-dependent protochlorophyllide oxidoreductases in aerobic anoxygenic phototrophs&#58; origin, phylogeny and function.</b> <i>Mol. Biol. Evol., Volume 38, Issue 3, Pages 819â€“837</i> <sup>e</sup>&nbsp;contributed equally</li>
                            </ul>
                    </div>
                </small>
            </div>
        
    </div>
</div>
<!--======================================================================-->
</div>
<!--======================================================================-->

